<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelect</class>
 <widget class="QWidget" name="ColorSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>169</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblRed">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="sliderRed">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinRed">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblGreen">
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="sliderGreen">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinGreen">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblBlue">
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="sliderBlue">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBlue">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLineEdit" name="txtColor">
      <property name="inputMask">
       <string>\#HHHHHH; </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinRed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinGreen</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBlue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderRed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderGreen</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderBlue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorSelect</receiver>
   <slot>colorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorSelect</receiver>
   <slot>colorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorSelect</receiver>
   <slot>colorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>colorChanged()</slot>
 </slots>
</ui>
